examples/test_metadata_loader/metadata.yaml

```diff
@@ -0,0 +1,8 @@
+document: test-metadata
+subtype: general
+enable_lint: True
+tag_filters:
+    - "docker_labels_inspect_test"
+import_tests:
+    - "*.py"
+    - "/bin/true"
```

Comment:
this is pretty interesting test

Reply:
yep, I were lazy to write any useful test, this dummy test is enough to schedule it via avocado :-)